build.gradle.kts

```
@@ -62,7 +62,7 @@ dependencies {
 
     runtimeOnly("org.slf4j:slf4j-nop:latest.release")
     implementation("org.apache.commons:commons-lang3:latest.release")
-    implementation("com.konghq:unirest-java:latest.release")
+    implementation("com.konghq:unirest-java:3.14.2") //to support Java 8
```

Typically comments follow the format of `// <comment>` with a space between the `//` and the comment itself. On top of being more consistent doing it this way, it's also more readable. I would strongly encourage you to format comments like that in the future.

```suggestion
    implementation("com.konghq:unirest-java:3.14.2") // to support Java 8
```